docs/dev/conf.py

```diff
@@ -19,7 +19,7 @@
 REPO_ROOT = THIS_DIR.parent.parent
 
 project = "MongoDB C Driver Development"
-copyright = "2024, MongoDB Inc"
+copyright = "2024, MongoDB, Inc"
```

Comment:
Any particular reason for leaving the year as 2024?

Reply:
Good catch. Updated.